tb_plugin/torch_tb_profiler/run.py

```diff
@@ -539,6 +540,26 @@ def process_modules_stats(parent, modules_stats):
         process_modules_stats(result["data"], self.module_stats)
         return result
 
+    def get_operator_tree(self):
+        root = next(iter(self.tid2tree.values()))
+
+        result = []
+
+        def traverse_node(parent, node):
+            d = {
+                "name": node.name,
+                "start_time": node.start_time,
+                "end_time": node.end_time,
+                "type": node.type,
+                "tid": node.tid,
+                "children": []
+            }
+            parent.append(d)
+            for child in node.children:
+                traverse_node(d["children"], child)
+        traverse_node(result, root)
+        return result[0]
```

Comment:
only return the first tree?

Reply:
there is only one item in the result array